hoot-core/src/main/cpp/hoot/core/conflate/DiffConflator.cpp

```diff
@@ -168,21 +168,21 @@ void DiffConflator::storeOriginalIDs(OsmMapPtr& pMap)
 {
   // Nodes
   const NodeMap &nodes = pMap->getNodes();
-  for (auto it = nodes.begin(); it != nodes.end(); ++it)
+  for (HashMap<long, NodePtr>::const_iterator it = nodes.begin(); it != nodes.end(); ++it)
```

Comment:
Last little nitpick for this, you already have a `typedef` for this:
```
typedef HashMap<long, NodePtr> NodeMap
```
I would suggest using the `typedef` here and in the two loops below.